Makefile

```diff
@@ -413,14 +460,17 @@ test-e2e-local-migration: $(GINKGO)
 test-e2e-local: $(GINKGO)
 	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="default"
 
+test-e2e-local-ha: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "simple || (high-availability && upgrade-to-$(SHOOT_FAILURE_TOLERANCE_TYPE))"
+
 ci-e2e-kind: $(KIND) $(YQ)
 	./hack/ci-e2e-kind.sh
 
-ci-e2e-kind-ha-node: $(KIND) $(YQ)
-	SHOOT_FAILURE_TOLERANCE_TYPE=node ./hack/ci-e2e-kind-ha.sh
+ci-e2e-kind-ha-single-zone: $(KIND) $(YQ)
+	SHOOT_FAILURE_TOLERANCE_TYPE=node ./hack/ci-e2e-kind-ha-single-zone.sh
 
-ci-e2e-kind-ha-zone: $(KIND) $(YQ)
-	SHOOT_FAILURE_TOLERANCE_TYPE=zone ./hack/ci-e2e-kind-ha.sh
+ci-e2e-kind-ha-multi-zone: $(KIND) $(YQ)
+	SHOOT_FAILURE_TOLERANCE_TYPE=zone ./hack/ci-e2e-kind-ha-multi-zone.sh
```

Comment:
My suggestion here was to set the `SHOOT_FAILURE_TOLERANCE_TYPE` environment variable in the respective `ci-e2e-kind-ha-*-zone.sh` scripts now that we have dedicated scripts for both types. No need to set it here in the `Makefile`, or?

Reply:
Moved `SHOOT_FAILURE_TOLERANCE_TYPE` to corresponding make target `test-e2e-local-ha-single-zone`/`test-e2e-local-ha-multi-zone`